src/EPR.CommonDataService.Api.UnitTests/Controllers/ApiControllerBaseTests.cs

```diff
@@ -0,0 +1,243 @@
+using EPR.CommonDataService.Api.Configuration;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.Extensions.Options;
+
+namespace EPR.CommonDataService.Api.UnitTests.Controllers;
+
+public class StubProblemDetailsFactory : ProblemDetailsFactory
+{
+    private readonly ValidationProblemDetails _validationProblemDetails;
+
+    public StubProblemDetailsFactory(ValidationProblemDetails validationProblemDetails)
+    {
+        _validationProblemDetails = validationProblemDetails;
+    }
+
+    public override ValidationProblemDetails CreateValidationProblemDetails(
+        HttpContext httpContext,
+        ModelStateDictionary modelStateDictionary,
+        int? statusCode = null,
+        string title = null,
+        string type = null,
+        string detail = null,
+        string instance = null)
+    {
+        return _validationProblemDetails;
```

Comment:
Could you add a comment to explain why this method has several parameters but they are not used?